commandline/src/main/java/tech/pegasys/eth2signer/commandline/Eth2SignerCommand.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.eth2signer.commandline;
+
+import tech.pegasys.eth2signer.core.Config;
+import tech.pegasys.eth2signer.core.Eth2Signer;
+
+import java.net.InetAddress;
+import java.nio.file.Path;
+
+import com.google.common.base.MoreObjects;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.HelpCommand;
+import picocli.CommandLine.Option;
+
+@SuppressWarnings("FieldCanBeLocal") // because Picocli injected fields report false positives
+@Command(
+    description =
+        "This command runs the EthSigner.\n"
+            + "Documentation can be found at https://docs.ethsigner.pegasys.tech.",
+    abbreviateSynopsis = true,
+    name = "ethsigner",
+    sortOptions = false,
+    mixinStandardHelpOptions = true,
+    versionProvider = VersionProvider.class,
+    header = "Usage:",
+    synopsisHeading = "%n",
+    descriptionHeading = "%nDescription:%n%n",
+    optionListHeading = "%nOptions:%n",
+    footerHeading = "%n",
+    subcommands = {HelpCommand.class},
+    footer = "EthSigner is licensed under the Apache License 2.0")
+public class Eth2SignerCommand implements Config, Runnable {
+
+  private static final Logger LOG = LogManager.getLogger();
+
+  @Option(
+      names = {"--data-path"},
+      description = "The path to a directory to store temporary files",
+      paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP,
+      arity = "1")
+  private Path dataPath;
+
+  @Option(
+      names = {"--key-store-path"},
+      description = "The path to a directory storing toml files defining available keys",
+      paramLabel = DefaultCommandValues.MANDATORY_PATH_FORMAT_HELP,
+      arity = "1")
+  private Path keyStorePath = Path.of("./");
+
+  @Option(
+      names = {"--logging", "-l"},
+      paramLabel = "<LOG VERBOSITY LEVEL>",
+      description =
+          "Logging verbosity levels: OFF, FATAL, WARN, INFO, DEBUG, TRACE, ALL (default: INFO)")
+  private final Level logLevel = Level.INFO;
+
+  @SuppressWarnings("FieldMayBeFinal") // Because PicoCLI requires Strings to not be final.
+  @Option(
+      names = {"--http-listen-host"},
+      description = "Host for JSON-RPC HTTP to listen on (default: ${DEFAULT-VALUE})",
```

Comment:
remove JSON-RPC, there are no RPCs for this app

Reply:
done